pkg/daemon/config.go

```diff
@@ -77,6 +77,8 @@ func WithLogLevel(logLevel string) NewDaemonOpt {
 	}
 }
 
+// In Shared mode, it is the mountpoint where single nydusd mounts fusedev. It has nothing
+// with fscache driver.
 func WithRootMountPoint(rootMountPoint string) NewDaemonOpt {
```

Comment:
We cannot assume that the fscache driver does not support multiple daemon mode, it is better not to bind the mountpoint implementation with the meaning of daemon mode.

Reply:
fixed